test/helpers/image_factory.go

```diff
@@ -42,6 +42,25 @@ func (i *ImageFactory) ImageDigest(imgRef string) string {
 	return digest.String()
 }
 
+// GetImageLayerDigest will return image's layer digest
+func (i *ImageFactory) GetImageLayerDigest(image string) []string {
```

Comment:
```suggestion
func (i *ImageFactory) GetFirstLayerDigest(image string) []string {
```

Reply:
Now we are returning all layers hence will rename it to `GetImageLayersDigest`